models/endpoints-model.js

```diff
@@ -0,0 +1,4 @@
+const endpointsJSON = require("../endpoints.json");
+exports.selectEndpoints = () => {
+  return endpointsJSON;
+};
```

Comment:
There's no need for a model in this endpoint as there's no database interaction (through sql), so you can do this whole thing within the controller!